docs/services/slam/_index.md

```diff
@@ -70,3 +64,18 @@ Go to your machine's **CONNECT** tab on the [Viam app](https://app.viam.com) and
 {{% /alert %}}
 
 {{< readfile "/static/include/services/apis/generated/slam.md" >}}
+
+## SLAM mapping best practices
+
+The best way to improve map quality is by taking extra care when creating the initial map.
+While in a slam session, you should:
+
+- turn gently and gradually, completely avoiding sudden quick turns
+- make frequent loop closures, arriving back at a previously mapped area so the machine can correct for errors in the map layout
+- stay relatively (but not extremely) close to walls
+- use a machine that can go smoothly over bumps and transitions between flooring areas
+- drive at a moderate speed
+- when using a wheeled base, try to include an [odometry movement sensor](/components/movement-sensor/wheeled-odometry). This helps the SLAM algorithm keep track of where the machine is moving.
+  - it is important to note that the [gyro-mpu6050](/components/movement-sensor/mpu6050) on the viam-rover **will not** satisfy the movement sensor requirement.
```

Comment:
Is it the gyro-mpu6050 (on the viam-rover v2) that reports AngularVelcoity and LinearAcceleration that cartographer can't take for odometry? I thought the imu code was just looking for those two movement primitives. I was under the impression that the axl345 accelerometer, that just reports LinearAcceleration (on the viam rover v1) that was the problematic one.

https://github.com/viamrobotics/viam-cartographer/blob/bf57db550f1b8587e8f082982533fa6c53f9c56d/sensors/movementsensor.go#L157

Reply:
didn't know there was a difference between v1 and v2 rovers and i grabbed the first sensor I saw, will update accordingly